app/helpers/application_helper.rb

```diff
@@ -147,7 +147,7 @@ def logo_svg
   end
 
   def community_qualified_name
-    "The #{ApplicationConfig['COMMUNITY_NAME']} Community"
+    "#{ApplicationConfig['COMMUNITY_NAME']} Community"
```

Comment:
As per https://dev-internal.slack.com/archives/CSYHV3NJ2/p1585232678024200

Reply:
Maybe `"#{community_name} Community"`. especially if we decide to add the caching to the former.